configs/components/hiera.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/hiera.git", "ref": "3047016da65d12ff7e316db17e40adec9a080090"}
+{"url": "git://github.com/puppetlabs/hiera.git", "ref": "refs/tags/3.1.0"}
```

Comment:
Re tags: 3.1.0 is correct. There is no 3.1.5 tag in hiera (although there is in facter).

Re commits: https://github.com/puppetlabs/hiera/commit/3047016da65d12ff7e316db17e40adec9a080090 is in the hiera repo, and is included in hiera's 3.1.0 tag.